testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java

```diff
@@ -0,0 +1,241 @@
+                appendln(whatIDidReport, (new Date()).toString());
+                appendln(whatIDidReport, appDir.getAbsolutePath());
+                appendlnSection(whatIDidReport, String.join(" ", cmd));
+
+                LOGGER.info("Building (" + cmd + ")");
+                buildService.submit(new Commands.ProcessRunner(appDir, buildLogA, cmd, 20));
+
+                buildService.shutdown();
+                buildService.awaitTermination(30, TimeUnit.MINUTES);
+
+                assertTrue(buildLogA.exists());
+                checkLog(cn, mn, app, mvnCmds, buildLogA);
+            }
+
+            // Run
+            runLogA = new File(logsDir + File.separator + subdir +  "-" + mvnCmds.name().toLowerCase() + "-run.log");
+            cmd = getRunCommand(mvnCmds.mvnCmds[mvnCmds != MvnCmds.DEV ? 1 : 0]);
+            LOGGER.info("Running (" + cmd + ")");
+            pA = runCommand(cmd, appDir, runLogA);
+
+            appendln(whatIDidReport, appDir.getAbsolutePath());
+            appendlnSection(whatIDidReport, String.join(" ", cmd));
+
+            // Test web page
+            LOGGER.info("Testing web page content...");
+            int timeout = mvnCmds != MvnCmds.DEV ? 5 : 20;
+            for (String[] urlContent : app.urlContent.urlContent) {
+                WebpageTester.testWeb(urlContent[0], timeout, urlContent[1], false);
+            }
+
+            processStopper(pA, false);
+        } finally {
+            // Make sure processes are down even if there was an exception / failure
+            if (pA != null) {
+                processStopper(pA, true);
+                pA.waitFor();
+            }
+
+            // Archive logs
+            if (buildLogA != null) {
+                archiveLog(cn, mn, buildLogA);
+            }
+            if (runLogA != null) {
+                archiveLog(cn, mn, runLogA);
+            }
+            writeReport(cn, mn, whatIDidReport.toString());
+
+            // Remove dir with special chars
+            if (appDestDir.exists()) {
+                FileUtils.deleteDirectory(appDestDir);
+            }
+        }
+    }
+
+
+    @Test
+    public void spacesJVM(TestInfo testInfo) throws IOException, InterruptedException {
+        testRuntime(testInfo, Apps.JAX_RS_MINIMAL, MvnCmds.JVM, "s p a c e s");
+    }
+
+    @Test
+    public void spacesDEV(TestInfo testInfo) throws IOException, InterruptedException {
+        testRuntime(testInfo, Apps.JAX_RS_MINIMAL, MvnCmds.DEV, "s p a c e s");
+    }
+
+    @Test
+    @Tag("native")
+    public void spacesNative(TestInfo testInfo) throws IOException, InterruptedException {
+        testRuntime(testInfo, Apps.JAX_RS_MINIMAL, MvnCmds.NATIVE, "s p a c e s");
+    }
+
+    @Test
+    public void specialJVM(TestInfo testInfo) throws IOException, InterruptedException {
+        testRuntime(testInfo, Apps.JAX_RS_MINIMAL, MvnCmds.JVM, ",;~!@#$%^&()");
+    }
+
+    @Test
+    @DisabledOnOs({OS.WINDOWS})
```

Comment:
Add a link for the reason of disabling it
```suggestion
    @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/9707
```